### PR TITLE
Update docs to include macOS/Windows dual boot workaround

### PR DESCRIPTION
**Pull request by tominabox1**
This is a workaround to allow you to use a bluetooth device on macOS and Windows from a single keyboard profile. I have tested this on Big Sur and Windows 10 successfully. This is distilled from various reference sources but primarily [insanelymac](https://www.insanelymac.com/forum/topic/268837-dual-boot-bluetooth-pairing-solved/)

See the updated page here: https://deploy-preview-549--zmk.netlify.app/docs/troubleshooting/

### CODE REVIEW

**Comment by petejohanson** [comment]
@tominabox1 Looks like prettier is unhappy about some changes. You can run `npm run prettier:format` from the `docs/` directory to have it fix it up for you.

File: docs/docs/troubleshooting.md [hunk 1]
```diff
@@ -73,3 +76,88 @@ CONFIG_BT_CTLR_TX_PWR_PLUS_8=y
...
+
+   | ![BT Plist](../docs/assets/troubleshooting/dualboot/plist.png) |
+   | :------------------------------------------------------------: |
+   |                        Bluetooth .plist                        |
+
+1. Look for the section that starts with `SMPDistributionKeys =`. Below this, find your host's bluetooth address (`65-FD-3B-1A-DB-48` in this example) and then look for the address of your keyboard (`E6-24-54-08-BC-A3` in this example).
+1. Under your keyboard address, find the value labeled LocalLTK (in the screenshot above it is `0x8968239e350b2cb7df16d8f47c774e2c`). Copy this value into a text file that you can access from Windows, this is the value we will need to enter into the registry later on.
+1. Reboot into Windows
+1. Once logged in, turn off bluetooth.
+1. Grab a piece of software called psexec (google) and copy the .exe to the desktop. This will allow us to easily open Regedit with elevated privelages.
```
Comment (Nicell) [inline docs/docs/troubleshooting.md:129]: Could we make a direct link here rather than saying to google something?

**Review by Nicell — CHANGES_REQUESTED** [review]
Overall this is very detailed with nice screenshots along the way. Look good! I have one small comment, and then I'm wondering more about the general structure of our troubleshooting page.

Our troubleshooting page is getting quite long, and I'm wondering if we should break up the sections into their own separate docs or possibly we could move this particular guide out into a new guide section? This could be in a separate PR, but this page is no doubt becoming quite long.

**Comment by karnadii** [comment]
I have tested this and worked on my dual boot.

**Comment by tominabox1** [comment]
> 
> 
> I have tested this and worked on my dual boot.

Thanks karnadii! I'll try to remember to add the requested link to the tool and push this on for final review tonight.

File: docs/docs/troubleshooting.md (hunk 1 above)
Comment (tominabox1) [reply docs/docs/troubleshooting.md]: I've added direct link in latest push. I don't care where the documentation ends up. I wasn't really sure what the best place for it was. Open to moving it wherever makes sense.

**Review by petejohanson — APPROVED** [review]
Agreed on @Nicell general comments on this page growing. I think getting this in now is fine, but we'll want to consider splitting this up into different sections, perhaps one for "build/flash" troubleshooting, another for runtime troubleshooting connections, split pairing, etc?

**Review by Nicell — APPROVED** [review]

**Comment by petejohanson** [comment]
@tominabox1 After some discussion, can we add a big warning block at the start of this, pointing out potential risks w/ making edits to your registry without being extra careful, etc?

**Comment by tominabox1** [comment]
Any merge available?

**Comment by karnadii** [comment]
Success doing a triple-boot (windows 10, macos monterey, manjaro) with my keyboard connection,
also update for macos big sur and monterey from https://github.com/profzei/Matebook-X-Pro-2018/wiki/Keep-Bluetooth-devices-paired-on-macOS-and-Windows
the windows part still the same,
for linux I just edit `[SlaveLongTermKey]` in the file `/var/lib/bluetooth/MAC_ADD/MAC_ADD/info`.
https://unix.stackexchange.com/questions/646012/bluetooth-pairing-in-windows-and-manjaro-arch-dual-boot

File: docs/docs/troubleshooting.md [hunk 2]
```diff
@@ -73,3 +76,96 @@ CONFIG_BT_CTLR_TX_PWR_PLUS_8=y
...
+   | ![Registry Key Edit](../docs/assets/troubleshooting/dualboot/Registry2.png) |
+   | :-------------------------------------------------------------------------: |
+   |                             Registry Key Editor                             |
+
+1. Select OK, exit Regedit, and reboot into Windows.
+1. Once rebooted, turn bluetooth back on, and make sure your keyboard is on the same profile you had selected in macOS. Your keyboard should connect and start working!
+1. Finally reboot into macOS and verify that your keyboard still works on the same profile here as well.
+
+If your board still does not work in one OS or the other, start over at the beginning with a clean pairing in each OS.
+git e
```
Comment (elsbrock) [inline docs/docs/troubleshooting.md:171]: remove?

**Comment by caksoylar** [comment]
I'll close this now that #2179 is in, thanks for the effort to compile this information.

**Pull request closed without merging**